src/DGmethods/DGmodel.jl

```diff
@@ -242,6 +242,47 @@ function nodal_update_aux!(f!, dg::DGModel, m::BalanceLaw, Q::MPIStateArray,
           knl_nodal_update_aux!(m, Val(dim), Val(polyorder), f!,
                           Q.data, dg.auxstate.data, dg.diffstate.data, t,
                           topology.realelems))
+
+end
+
+"""
+    cfl(wavespeed_dt, dg::DGModel, m::BalanceLaw, Q::MPIStateArray,
```

Comment:
A detail: I wouldn't call this wavespeed. Most of the time, it will not be a wavespeed but an advective velocity (e.g., jet speed).

Reply:
I generalized this callback so that it computes the local cfl constant directly given the local minimum node spacing Δx.  This should allow for the computation of hyperbolic and diffusive cfl constants.